src/_guides/language/analysis-options.md

```diff
@@ -239,7 +239,34 @@ include: package:pedantic/analysis_options.yaml
   or [disable individual rules][].
 {{site.alert.end}}
 
-[disable individual rules]: #disabling-individual-rules
+### Enabling the set of rules corresponding to the Effective Dart guide: effective_dart {#enabling-effective-dart-rules}
+
+To enable the list of linter rules corresponding to the Effective Dart guide, add a dev dependency on the [effective_dart package]({{site.pub-pkg}}/effective_dart) and include its `analysis_options.yaml` file.
```

Comment:
We use 80-or-fewer-character lines, with breaks on phrases (aka _semantic line breaks_): https://github.com/dart-lang/site-shared/wiki/Writing-for-Dart-and-Flutter-websites

Also, let's link to Effective Dart here and move the package link to the bottom of the file.
Finally... maybe we can remove the text about analysis_options.yaml from the first paragraph,
since there's no sentence between it and the example, as there is in the previous section.
Something like this:

```
To enable linter rules corresponding to the guidelines in [Effective Dart][],
add a dev dependency on the [effective_dart package][]:

...
[effective_dart package]: {{site.pub-pkg}}/effective_dart```